stubs/netaddr/netaddr/eui/__init__.pyi

```diff
@@ -40,7 +41,7 @@ class EUI(BaseIdentifier):
     @property
     def value(self) -> int: ...
     @value.setter
-    def value(self, value: str | SupportsInt | SupportsIndex) -> None: ...
+    def value(self, value: ConvertibleToInt) -> None: ...
```

Comment:
This seems like a good change, since the `.value` property is explicitly there in order to transform `str`, `bytes` objects etc. into `int` before they're stored on the `EUI` instance